docs/freebsd.md

```diff
@@ -0,0 +1,25 @@
+# FreeBSD
+
+
+| :zap:        FreeBSD runtimes are at the very early stage of development |
+|--------------------------------------------------------------------------|
+
+nerdctl provides experimental support for running FreeBSD jails.
+
+## Installation
+
+You will need the most up-to-date containerd build along with a containerd shim,
+such as [runj](https://github.com/samuelkarp/runj). Follow the build
+instructions in the respective repositories.
+
```

Comment:
Could you show full `nerdctl run` example?

`nerdctl run --some-required-flags SOME-PUBLIC-IMAGE echo hello`